db/api/api.js

```diff
@@ -4,12 +4,17 @@ const {
   handleCustomErrors,
   handleBadPaths,
 } = require("../api/controllers/handleErrors");
-const { getCategories, getReviews } = require("./controllers/index");
+const {
+  getCategories,
+  getReviews,
+  getReviewIdComments,
+} = require("./controllers/index");
 const app = express();
 
 app.get("/api/categories", getCategories);
 app.get("/api/reviews", getReviews);
 app.get("/api/reviews/:review_id", getReviews);
```

Comment:
definitely should be two different controllers because we're getting back two different responses